jujugui/static/gui/src/app/assets/svgs/icon-social-google.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm assuming these are minified versions of the old icons, but just FYI the new svg sprite system actually does this for us!